src/Containers/Microsoft.NET.Build.Containers/Tasks/ComputeDotnetBaseImageTag.cs

```diff
@@ -99,7 +99,7 @@ public override bool Execute()
                 }
                 Log.LogError(Resources.Strings.InvalidSdkPrereleaseVersion, channel);
                 return null;
-            case "alpha" or "dev" or "ci":
+            case "alpha" or "dev" or "ci" or "rtm":
```

Comment:
this change doesn't do anything - see line 93 above.

Reply:
Oh 🙂 You're right; I can remove it